dropwizard-app/src/test/java/org/coner/core/ConerCoreServiceTest.java

```diff
@@ -28,9 +30,11 @@
 public class ConerCoreServiceTest {
 
     @Mock
-    private EventGateway eventGateway;
+    EventGateway eventGateway;
```

Comment:
Why remove the private modifier?  Also, why did you have to call initMocks() in the ConerDropwizardApplicationTest but not in this one.  I thought when you use @Mock you have to call initMocks or use the Mockito junit runner (or use the stacking mock(Class.class);).